File: bank-service/src/main/java/rs/edu/raf/repository/accounts/BusinessBankAccountRepository.java
package rs.edu.raf.repository.accounts;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import rs.edu.raf.model.accounts.BusinessBankAccount;

@Repository
public interface BusinessBankAccountRepository extends JpaRepository<BusinessBankAccount,Long> {

}

File: bank-service-kotlin/src/main/kotlin/org/example/bankservicekotlin/repository/accounts/BusinessBankAccountRepository.kt
package org.example.bankservicekotlin.repository.accounts

import org.example.bankservicekotlin.model.accounts.BusinessBankAccount
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface BusinessBankAccountRepository: JpaRepository<BusinessBankAccount, Long> {
}